soc/espressif/esp32c3/default.ld

```diff
@@ -641,23 +641,87 @@ SECTIONS
 
   /* --- END OF DRAM --- */
 
+  /* --- START OF .flash.text --- */
+
+  .flash.text_dummy (NOLOAD):
+  {
+    . = ALIGN(CACHE_ALIGN);
+  } GROUP_LINK_IN(ROMABLE_REGION)
```

Comment:
Should it be `FLASH_CODE_REGION` instead of `ROMABLE_REGION`?

Reply:
> Should it be `FLASH_CODE_REGION` instead of `ROMABLE_REGION`?

Nope, the purpose here is to align the flash location counter, which is random after spilling the previous loadable sections. The `FLASH_CODE_REGION`, at this time, is unused and aligned.